infra/main.bicepparam

```diff
@@ -1,4 +1,8 @@
 using 'main.bicep'
 
-param environmentName = readEnvironmentVariable('AZURE_ENV_NAME', '')
+param environmentName = readEnvironmentVariable('AZURE_ENV_NAME', 'drda')
```

Comment:
what is drda?

Reply:
I think it is just an abbreviation for `demystifying-rag-dotnet to avm`, but it is just a placeholder because `AZURE_ENV_NAME` is a `Required` parameter for the user, so the default value will not be used.

Reply:
can we remove it?